backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/service/CreateOrderCandidateCommand.java

```diff
@@ -98,6 +99,11 @@ public I_PP_Order_Candidate execute()
 		ppOrderCandidateRecord.setC_OrderLine_ID(OrderLineId.toRepoId(request.getSalesOrderLineId()));
 		ppOrderCandidateRecord.setM_ShipmentSchedule_ID(ShipmentScheduleId.toRepoId(request.getShipmentScheduleId()));
 
+		final I_PP_Product_Planning productPlanning = productPlanningsRepo.getById(request.getProductPlanningId());
+		final BigDecimal qtyProcessed_OnDate = productPlanning.getQtyProcessed_OnDate();
```

Comment:
this is kindow a shortcut..but i guess it's OK..mabe we need to go the full circle via material-dispo later though, wen we better understand how the two columns are used